pkg/simple/client/logging/elasticsearch/api_body.go

```diff
@@ -119,14 +119,15 @@ func (bb *bodyBuilder) mainBool(sf logging.SearchFilter) *bodyBuilder {
 		ms = append(ms, Match{Bool: &b})
 	}
 
-	if !sf.Starttime.IsZero() || !sf.Endtime.IsZero() {
-		fromTo := Match{
-			Range: &Range{&Time{
-				Gte: &sf.Starttime,
-				Lte: &sf.Endtime,
-			}},
-		}
-		ms = append(ms, fromTo)
+	r := &Range{Time: &Time{}}
```

Comment:
please don't use single letter named variable, it's not a good practice. Use meaningful names.

Reply:
It's a local variable. Besides, range is keyword

![image](https://user-images.githubusercontent.com/22350668/83341996-547b4d00-a31c-11ea-9e48-c7ffa9240bc8.png)

![image](https://user-images.githubusercontent.com/22350668/83342001-63fa9600-a31c-11ea-9261-6c323acd77e0.png)